gbpservice/neutron/tests/unit/nfp/configurator/agents/test_vpn_agent.py

```diff
@@ -11,28 +11,52 @@
 #    under the License.
 
 
-from gbpservice.neutron.tests.unit.nfp.configurator.test_data import \
-                                                                vpn_test_data
+import mock
+
+from gbpservice.neutron.tests.unit.nfp.configurator.test_data import (
+    vpn_test_data)
 from gbpservice.nfp.configurator.agents import vpn
-from gbpservice.nfp.configurator.drivers.vpn.vyos import vyos_vpn_driver
+from gbpservice.nfp.configurator.lib import vpn_constants as const
 
-import mock
-import unittest
+from neutron.tests import base
+
+
+class VPNaasRpcManagerTestCase(base.BaseTestCase):
+    '''
+    Implements test cases for RPC manager methods of vpn agent
+    '''
+    def __init__(self, *args, **kwargs):
+        super(VPNaasRpcManagerTestCase, self).__init__(*args, **kwargs)
+        self.dict_obj = vpn_test_data.VPNTestData()
```

Comment:
Avoid obj in variable names. Also, name this variable appropriately. dict_obj is more generic name.